### PR TITLE
Created API scopes landing page in publisher.

### PR DESCRIPTION
**Pull request by prasa7**
## Purpose
Created API scopes landing page in publisher

### CODE REVIEW

**Comment by tmkasun** [comment]
doc impact: Need to update the screenshots in API documentation related sections

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx [hunk 1]
```diff
@@ -170,14 +159,42 @@ class Scopes extends React.Component {
...
+                            <CardContent>
+                                <Typography align='justify' component='p'>
+                                    <FormattedMessage
+                                        id='create.scope.description'
+                                    />
+                                </Typography>
+                            </CardContent>
+                            <CardActions>
+                                <Button variant="contained" color="primary" className={classes.button}>
+                                    Crearte scopes
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx]: typo

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx [hunk 2]
```diff
@@ -170,14 +159,42 @@ class Scopes extends React.Component {
...
-            return <Progress />;
+        if (scopes.length === 0) {
+            return (
+                <Grid container justify='center'>
+                    <Grid item sm={5}>
+                        <Card className={classes.card}>
+                            <Typography align='justify' className={classes.headline} gutterBottom variant='headline' component='h2'>
+                                <FormattedMessage
+                                    id='create.scopes'
+                                    defaultMessage='Create Scopes..'
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx]: i18n?

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx [hunk 3]
```diff
@@ -22,8 +22,23 @@ import PropTypes from 'prop-types';
...
+import Divider from '@material-ui/core/Divider';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
 
+const styles = theme => ({
+    buttonProgress: {
+        position: 'relative',
+        margin: theme.spacing.unit,
+    },
+    headline: { paddingTop: '5px', paddingLeft: '10px' },
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx]: should we use variables instead?

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx [hunk 4]
```diff
@@ -170,14 +159,42 @@ class Scopes extends React.Component {
...
     render() {
-        const { apiScopes } = this.state;
+        const { api } = this.props;
+        const { scopes } = api;
+        const { classes } = this.props;
 
-        if (this.state.notFound) {
-            return <ResourceNotFound message={this.props.resourceNotFountMessage} />;
+        if (!scopes) {
+            return <Progress />;
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx]: Can we have a FormattedMessage ?

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx [hunk 5]
```diff
@@ -170,14 +159,42 @@ class Scopes extends React.Component {
...
+                                <FormattedMessage
+                                    id='create.scopes'
+                                    defaultMessage='Create Scopes..'
+                                />
+                            </Typography>
+                            <Divider />
+                            <CardContent>
+                                <Typography align='justify' component='p'>
+                                    <FormattedMessage
+                                        id='create.scope.description'
```
[the remark this answers is not in the document]
Comment (prasa7) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx]: Added default message

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx (hunk 4 above)
Comment (prasa7) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scopes.jsx]: Added FormattedMessage

**Pull request merged**